packages/cli/src/index.ts

```diff
@@ -144,12 +144,6 @@ const main = async () => {
     return 1;
   }
 
-  let cwd = argv['--cwd'];
-  if (cwd) {
-    process.chdir(cwd);
-  }
-  cwd = process.cwd();
```

Comment:
Don't we need to keep this here so that `await mkdirp(VERCEL_DIR)` uses the correct cwd?

Reply:
No, `VERCEL_DIR` is the global dir.